applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.cpp

```diff
@@ -1304,4 +1304,16 @@ void ShellCrossSection::Ply::RecoverOrthotropicProperties(const IndexType IdxCur
     laminaProps[SHELL_ORTHOTROPIC_LAYERS] = current_ply_properties;
 }
 
+
+std::vector<ConstitutiveLaw::Pointer> ShellCrossSection::GetConstitutiveLawsVector(const Properties& rProps)
+{
+    std::vector<ConstitutiveLaw::Pointer> claws_vector;

```

Comment:
I woul assign size, it is known a a priori

Reply:
not directly like in the solids, as the section can have multiple layers
Hence I think it is not worth the effort of looping twice, esp since this fct is not called in performance critical parts of the code

Reply:
I mean I could do it as above, but I really think it is not worth the effort ...

Reply:
Removing my block